docs/develop/yeoman-generator-overview.md

```diff
@@ -65,7 +65,7 @@ The next question asks you to choose between **TypeScript** and **JavaScript**.
 
 ![Screenshot showing that the user chose "Office Add-in Task Pane project" to the preceding question and shows the prompt for language, and the possible answers, TypeScript and JavaScript, in the Yeoman generator.](../images/yo-office-language-prompt.png)
```

Comment:
Nit: Remove "screenshot" from alt text (here and other images in the article).

Reply:
I'll create a separate issue to scrub this later, since this PR doesn't touch the screenshot.